llmproxy/provider/google/vertexai.py

```diff
@@ -30,6 +37,18 @@
             "Financial Task": 2,
             "Content Recommendation Task": 1,
         },
+        "gemini-pro": {
```

Comment:
I'm assuming this is random, or similar to GPT? You may want to ask Dinuja or do some research and comparison with GPT-4 modles

Reply:
yes random